src/registrar/templates/includes/form_messages.html

```diff
@@ -0,0 +1,11 @@
+{% if messages %}
+{% for message in messages %}
```

Comment:
**question**:This will show even messages that are tagged as "base", leading to duplicates with any that show up in `base.html`. Do you want to only show messages here with `'base' not in message.extra_tags`?

Reply:
Right now my understanding of base messages is for debug only. My main concern if we do the above is, supposedly we add more tags to the extra_tags, that will lead to issues because of how I use the tag (success/caution) for CSS/USWDS styling. Something I plan to keep eye on during the project.